packages/cloud_firestore/cloud_firestore_web/lib/src/query_web.dart

```diff
@@ -0,0 +1,298 @@
+// Copyright 2017, the Chromium project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:cloud_firestore_platform_interface/cloud_firestore_platform_interface.dart';
+import 'package:firebase/firestore.dart' as web;
+
+import 'package:cloud_firestore_web/src/utils/document_reference_utils.dart';
+
+/// Web implementation for firestore [QueryPlatform]
+class QueryWeb extends QueryPlatform {
+  final web.Query _webQuery;
+  final FirestorePlatform _firestore;
+  final bool _isCollectionGroup;
+  final String _path;
+  final List<dynamic> _orderByKeys;
+  static const _kChangeTypeAdded = "added";
+  static const _kChangeTypeModified = "modified";
+  static const _kChangeTypeRemoved = "removed";
+
+  /// Builds an instance of [QueryWeb] using [_path] & [_webQuery]
```

Comment:
```suggestion
  /// Builds an instance of [QueryWeb] delegating to a package:firebase [Query]
```

Doc comments shouldn't reference private stuff.